### ISSUE TITLE
driverless no longer works for local queues due new ippfind call

### ISSUE THREAD

**Issue by zdohnal** [OpenPrinting/cups-filters#294]
Hi,

we have ```driverless``` simple sanity test (just ) within our Fedora/RHEL CI. Because we don't have any IPP everywhere supported device in our lab, I wrote the test using previously created local cups queue:

```
# lpadmin -p test -v file:/tmp/ps -m drv:///sample.drv/generic.ppd -E
```

and then call ```driverless```  on its device uri:

```
# driverless ipp://localhost:631/printers/test > out.ppd
```

With 1.28.0, ```ippfind``` is called within ```get_printer_attributes5()```, which searches for devices on the local network meaning it doesn't count a local queue - then whole ```driverless``` process ends with error.

I understand ```driverless``` binary is meant for communicating with a printer, but in my opinion it would be good to support local queues for simple testing purposes.

Would it be acceptable if I'll create '--local' parameter for ```driverless``` , which will make ```generate_ppd()``` to call ```get_printer_attributes3()``` (uses CUPS_BACKEND_URI_CONVERTER - original uri resolver)? Please let me know if it is possible or if you have an other idea.

**Comment by tillkamppeter** [comment]
I have already seen it. I will fix this soon. And it should work automatically with local services. So an extra option, like `--local` should be avoided.

**Comment by tillkamppeter** [comment]
Fixed in commit 8dd6a38e64.

**Comment by zdohnal** [comment]
The test passes now, thanks for the fix!
